e2e/libs/system_backup/rest.py

```
@@ -0,0 +1,80 @@
+
+    def cleanup_system_backups(self):
+        system_backups = get_longhorn_client().list_system_backup()
+        for system_backup in system_backups:
+            logging(f"Deleting system backup {system_backup['name']}")
+            try:
+                get_longhorn_client().delete(system_backup)
+            except Exception as e:
+                logging(f"Deleting system backup {system_backup['name']} failed: {e}")
+
+        deleted = False
+        for i in range(self.retry_count):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)